src/tests/ftest/container/query_attribute.py

```diff
@@ -47,6 +72,7 @@ def test_container_query_attr(self):
         :avocado: tags=all,full_regression
         :avocado: tags=vm
         :avocado: tags=container
+        :avocado: tags=daos_cmd
```

Comment:
"features" are usually put on the same line. For this and other files
```suggestion
        :avocado: tags=container,daos_cmd
```

But... Is the idea to add `daos_cmd` to EVERY test that uses the `daos` command? Because that be would nearly every test in the framework, since we use `daos` to create containers 99% of the time. Most references are just buried in the framework. In fact, files that directly reference `DaosCommand`, etc. are slowly being changed to use wrapper utils.

Reply:
> Is the idea to add daos_cmd to EVERY test that uses the daos command?

No, more just to find a set of tests that will exercise as much of the command surface as possible in order to verify that core changes to the command don't break existing functionality.

Reply:
FWIW, the `container` tag covers much of the `daos container` operations. If adding `daos_cmd` helps you / the control team, I'm okay with it. We still have a bit of work to do with tags, so it's hard to be picky :)

Reply:
There is other functionality being added that isn't covered by the `container` tag, e.g. datamover stuff, autotest, etc. The idea is to try to pick tests that we can't easily run in a single-node dev setup to verify that we haven't broken some important but obscure part of the tool.